data/mods/DinoMod/monsters/zed-dinosaur.json

```diff
@@ -343,7 +387,7 @@
       [ "SMASH", 30 ]
     ],
     "description": "Massive, long-necked, four-legged dinosaur corpse with a long, whip-like tail.  The head is upright and the neck looks like it would still make a good strong club.",
```

Comment:
```suggestion
    "description": "A massive, long-necked, four-legged dinosaur corpse with a long, whip-like tail.  The head is still upright and the neck looks like it would still make a good, strong club.",
```